### ISSUE TITLE
11.1.3.1d Layouttabellen nach WCAG erlaubt

### ISSUE THREAD

**Issue by johannesFischer84** [BIK-BITV/BIK-App-Test#14]
In der Prüfanleitung sollte deutlicher hervorgehoben werden, dass Layouttabellen nach den WCAG erlaubt sind, auch wenn sie nicht empfohlen werden. Die WCAG erlaubt nur keine Strukturinformationen von Tabellen bei Layouttabellen, siehe [Failure F46](https://www.w3.org/WAI/WCAG21/Techniques/failures/F46). An möglichen Strukturinformationen scheint es mir nur Tabellenüberschriftszellen unter nativem iOS zu geben. Tabellenbeschriftungen (`caption` in HTML) können meiner Erkenntnis nach sowieso nicht ausgezeichnet werden.

Ich würde mir folgende Ergänzungen wünschen:

## Was wird geprüft
Bisher: Tabellenstruktur-Auszeichnungen sollen nicht für Layouttabellen verwendet werden.
Neu: Tabellenstruktur-Auszeichnungen wie z. B. Überschriftszellen dürfen nicht für Layouttabellen verwendet werden.

## Prüfung Nummer 3
Bisher: Den Bereich navigieren bzw. antippen, der für eine semantische Tabelle gehalten wird. Prüfen ob der Screenreader den Zusatz „Tabelle“ ausgibt.
Neu zusätzlich: In diesem Fall darf z. B. bei einer Überschriftenzelle nicht der Zusatz "Überschrift" oder ähnlich ausgegeben werden.

## Bewertung nicht erfüllt
Bisher: Es werden Layouttabellen eingesetzt, der Screenreader gibt unnötige weitere Tabellendetails aus.
Neu: Bei Layouttabellen gibt der Screenreader unnötige weitere Tabellenstrukturen aus, wie z. B. eine Überschriftenzelle.

**Comment by detlevhfischer** [comment]
Danke @johannesFischer84 ich habe das im Prüfschritt ergänzt.

**Comment by johannesFischer84** [comment]
@detlevhfischer Danke, Detlev. Nur ist dabei die Überschrift "Bewertung" vor "nicht erfüllt" in die Zeile davor gerutscht (Nummer 4 der Prüfung) und nicht mehr als Überschrift erkennbar. Diesen Tippfehler kannst du bestimmt noch korrigieren.

**Comment by detlevhfischer** [comment]
Danke, ist korrigiert!
